style.css

```diff
@@ -0,0 +1,81 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&family=Ubuntu:wght@300&display=swap");
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  z-index: 99999999;
+  background-color: #fff;
+}
+
+.welc {
+  color: #172b4f;
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+}
+
+.menu {
+  color: #172b4f;
+  font-size: 12px;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  background-image: url("/images/header-illsutration-mobile@2x.jpg");
+  background-color: #fff;
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.identity {
+  color: #172b4f;
+  font-size: 40px;
+  font-weight: 700;
+  font-style: normal;
+  text-align: center;
+}
+
+.identity span {
+  color: #36b37e;
+}
+
+.profile {
+  font-weight: 400;
+  font-size: 20px;
+  color: #172b4d;
+  text-decoration: none;
+  margin-top: 20px;
+  line-height: 28px;
+  text-align: center;
+}
+
+.icon-group {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 0%;
+}
```

Comment:
- [ ] Kindly remove the dots from your unordered list to make it look more professional.
![image](https://user-images.githubusercontent.com/91486791/219731659-55139c29-d223-422a-9587-b7b8dcf7dc0d.png)